projects/hslayers/common/remove-multiple/remove-layer-dialog.component.ts

```diff
@@ -12,6 +12,12 @@ import {
 import {Layer} from 'ol/layer';
 import {getName, getTitle} from 'hslayers-ng/common/extensions';
 
+/**
+ * Both catalogue and mapcatalogue will remove layer from  map as well.
+ * Difference is with layer 'form' - string vs Layer<Source> (see service removeMultipleLayers overloads)
+ * String - layman layer descriptor. Not all layers are guaranted to be in map (mroe likely not)
+ * Layer<Source> - OL layer. Layers already in map
+ */
```

Comment:
```suggestion
/**
 * Both catalogue and mapcatalogue will remove layer from the map as well.
 * Difference is with layer 'form' - string vs Layer<Source> (see service removeMultipleLayers overloads)
 * string - Layman layer descriptor. Not all layers are guaranteed to be in map (more likely not)
 * Layer<Source> - OL layer. Layers already in map
 */
```

Beside the typos, what does it mean that `string` is a "Layman layer descriptor"? Descriptor is usually an object, no?

Reply:
Right . Precisely layman layer descriptor name property because for record is add-data catalogue only GET layers response is avilable.